src/components/Form.js

```diff
@@ -34,6 +34,7 @@ export default class Form extends Component {
         return props.uiSchema.renderCodeBlock({ node });
       };
     }
+    window.renderers = props.uiSchema.renderers;
```

Comment:
This doesn't look right. See my other comment about putting things on the window...

Reply:
We might not need to do this at all as I have suggested an alternate way to handle markdown rendering in another comment.